_plugins/server_generator.rb

```diff
@@ -1,5 +1,7 @@
 module Jekyll
   class ServerPage < Page
+    include Jekyll::ThanksToFilter
```

Comment:
If this really a filter if you're just going to include it to get a method?

It sounds more like it's just some library code...

I would either just add the `thanks_to` method as a private method in this file, or if you really want it separate then it needs a new name/location as the current name has a specific meaning in the context of jeykll.